src/main/java/school/hei/haapi/endpoint/event/model/CheckMobilePaymentTransactionTriggered.java

```diff
@@ -13,7 +13,7 @@
 public class CheckMobilePaymentTransactionTriggered extends PojaEvent {
   @Override
   public Duration maxConsumerDuration() {
-    return Duration.ofSeconds(30);
+    return Duration.ofSeconds(100);
```

Comment:
is it sure that 100 seconds ( 1minute 40) is enough to handle any CheckMobilePaymentTransactionTriggered call ?